src/components/internals/rich-text-body/rich-text-body.js

```diff
@@ -20,15 +22,95 @@ import {
   UnderlineIcon,
   RedoIcon,
   UndoIcon,
+  SubscriptIcon,
+  StrikethroughIcon,
+  SuperscriptIcon,
+  MoreStylesIcon,
 } from './icons';
+import Spacings from '../../spacings';
 import Button from './rich-text-body-button';
 import Divider from './divider';
-import MultiDropdown from './multi-dropdown';
 import Dropdown from './dropdown';
+import { DropdownItem as StyledDropdownItem } from './dropdown.styles';
 import { MARK_TAGS, BLOCK_TAGS } from '../rich-text-utils/tags';
 import hasBlock from '../rich-text-utils/has-block';
 import messages from './messages';
 
+const MoreStylesDropdownLabel = () => <MoreStylesIcon size="medium" />;
+MoreStylesDropdownLabel.displayName = 'MoreStylesDropdownLabel';
+
+const MoreStylesDropdownItem = props => {
```

Comment:
Can you elaborate why it is named like this. Sorry, can’t follow 😅

Reply:
We use the Dropdown twice, and each time needs "custom" components to match the design. Does that make sense?